core/divesitehelpers.cpp

```diff
@@ -53,158 +43,134 @@ void ReverseGeoLookupThread::run() {
 	request.setRawHeader("User-Agent", getUserAgent().toUtf8());
 	connect(&timer, SIGNAL(timeout()), &loop, SLOT(quit()));
 
-	Q_FOREACH (const GeoLookupInfo& info, geo_lookup_data ) {
-		struct dive_site *ds = info.uuid ? get_dive_site_by_uuid(info.uuid) : &displayed_dive_site;
+	struct dive_site *ds = &displayed_dive_site;
 
-		// first check the findNearbyPlaces API from geonames - that should give us country, state, city
-		request.setUrl(geonamesURL.arg(uiLanguage(NULL)).arg(info.lat.udeg / 1000000.0).arg(info.lon.udeg / 1000000.0));
+	// first check the findNearbyPlaces API from geonames - that should give us country, state, city
+	request.setUrl(geonamesURL.arg(uiLanguage(NULL)).arg(ds->latitude.udeg / 1000000.0).arg(ds->longitude.udeg / 1000000.0));
 
-		QNetworkReply *reply = rgl->get(request);
-		timer.setSingleShot(true);
-		connect(reply, SIGNAL(finished()), &loop, SLOT(quit()));
-		timer.start(5000);   // 5 secs. timeout
-		loop.exec();
+	QNetworkReply *reply = rgl->get(request);
+	timer.setSingleShot(true);
+	connect(reply, SIGNAL(finished()), &loop, SLOT(quit()));
+	timer.start(5000);   // 5 secs. timeout
+	loop.exec();
 
-		if(timer.isActive()) {
-			timer.stop();
-			if(reply->error() > 0) {
-				report_error("got error accessing geonames.org: %s", qPrintable(reply->errorString()));
-				goto clear_reply;
-			}
-			int v = reply->attribute(QNetworkRequest::HttpStatusCodeAttribute).toInt();
-			if (v < 200 || v >= 300)
-				goto clear_reply;
-			QByteArray fullReply = reply->readAll();
-			QJsonParseError errorObject;
-			QJsonDocument jsonDoc = QJsonDocument::fromJson(fullReply, &errorObject);
-			if (errorObject.error != QJsonParseError::NoError) {
-				report_error("error parsing geonames.org response: %s", qPrintable(errorObject.errorString()));
-				goto clear_reply;
-			}
-			QJsonObject obj = jsonDoc.object();
-			QVariant geoNamesObject = obj.value("geonames").toVariant();
-			QVariantList geoNames = geoNamesObject.toList();
-			if (geoNames.count() > 0) {
-				QVariantMap firstData = geoNames.at(0).toMap();
-				int ri = 0, l3 = -1, lt = -1;
-				if (ds->taxonomy.category == NULL) {
-					ds->taxonomy.category = alloc_taxonomy();
+	if(timer.isActive()) {
```

Comment:
space after `if`.

Reply:
This is a pure re-indent. I want `git show -w` to be empty so it's clear the commit only deals with whitespace.
I can add another commit that fixes this.